transactions/idTableStaking/get_node_total_commitment_without_delegators.cdc

```diff
@@ -0,0 +1,7 @@
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+
+// This script returns the balance of staked tokens of a node
+
+pub fun main(nodeID: String): UFix64 {
+    return FlowIDTableStaking.getNodeCommittedBalanceWithoutDelegators(nodeID)
+}
```

Comment:
I know it's a UNIX affectation but this and get_node_total_commitment.cdc don't have newlines at the end.